config/setupTests.js

```diff
@@ -0,0 +1,84 @@
+jest.mock('react', () => ({
+  ...jest.requireActual('react'),
+  useLayoutEffect: jest.requireActual('react').useEffect,
+}));
```

Comment:
Why this?

Reply:
aha will remove this of course. it is not needed